### DISCUSSION TITLE
Where to customize config

### DISCUSSION THREAD

**Discussion by ledmirage** [Jyers/Marlin#1202] [Q&A] [answered]
Hi there,

I have been using pre-built firmware for my Ender 3v2, so far they are working great, love the work done here by Jyers and other contributors.

I am planning to switch out my extruder to e3d titan to try out direct drive, my initial test shows the extruder will rotate in opposite direction with current firmware, so I would like to compile my own to invert the extruder direction.

I can see there are some config templates under

  Marlin/Configuration Files/E3V2 Templates/

I would like to check
- are these the configs used to compile for the released firmware?
- i am using BLTouch 5x5, but seems there is no such config?

I would like to use the same config as the pre-build firmware to avoid error when customizing.

Thanks in advance for your help!

**Comment by names12** [comment]
Hi! 
I recently had a similar issue. I wanted to use Jyers, but my own config. As far as I know, those are the templates that are used for the main releases. 
But here is what I did: I became a Patreon https://www.patreon.com/jyers and now I have access to Jyers shared server, where there is a very simple text doc that I answer a few questions, (one of which is the direction of the extruder!), save the document in the proper place and labeled what I want the .bin file to be. Every hour, the server compiles the latest Jyers Firmware with my configs based off that document. Takes the guess work out of it all. Works like a charm! 
You can also use these templates there. Totally worth it. 
Hope it helps!

**Reply by ledmirage** [reply]
thanks for the advise. i dun mind becoming a pateron but I do want to have the capability to do the complication myself so i have better control :)

**Comment by Cal-LF** [comment, marked as answer]
Start with the BL Touch 3X3 template and make your changes as needed. Change to 5X5 grid. Reverse the extruder rotation.
 #define GRID_MAX_POINTS_X 3    // Don't use more than 7 points per axis, implementation limited.
In Marlin Configuration.h you can switch the boolean option from FALSE to TRUE or vice versa. Key variable is #define INVERT_E0_DIR

**Reply by ledmirage** [reply]
thanks, i will give it a try!
